compiler/src/dotty/tools/dotc/transform/sjs/JSInteropUtils.scala

```diff
@@ -0,0 +1,78 @@
+package dotty.tools.dotc
+package transform
+package sjs
+
+import core._
+import util.SrcPos
+import Annotations._
+import Constants._
+import Contexts._
+import Decorators._
+import DenotTransformers._
+import Flags._
+import NameOps._
+import Names._
+import Phases._
+import Scopes._
+import StdNames._
+import Symbols._
+import SymDenotations._
+import SymUtils._
+import ast.Trees._
+import Types._
+
+import dotty.tools.backend.sjs.JSDefinitions.jsdefn
+
+object JSInteropUtils {
+  enum JSName {
+    case Literal(name: String)
+    case Computed(sym: Symbol)
+
+    def displayName(using Context): String = this match {
+      case Literal(name) => name
+      case Computed(sym) => sym.fullName.toString()
+    }
+  }
+
+  extension (sym: Symbol) {
```

Comment:
There is some duplication with the methods in `dotty.tools.backend.sjs.JSInterop`

Reply:
I've fixed that by making all the methods of `JSInterop` forward to the new ones in `JSInteropUtils`. Eventually `JSInterop` should disappear, but keeping it for now allows not to change everything in `JSCodeGen` right now for little reason.